nixos/default.nix

```diff
@@ -33,7 +35,7 @@
       listenAddr = mkOption {
         description = "Address to listen on";
         type = types.str;
-        default = ":443";
+        default = defaults.listenAddr;
```

Comment:
Hah, these default conversions are great. Thanks (:

Reply:
In the long run, we should probably convert those to be more auto generated and not duplicated but good enough for now.